build/metadata.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""
+Process manuscript metadata and author metadata
+"""
+import collections
+import pathlib
+
+import yaml
+
+
+def get_metadata_object(path):
+    """
+    Load all of the metadata from a yaml file, & return.
+
+    :param path: pathlib.Path to a metadata YAML file.
+    :return: structured metadata object.
+    """
+    class metadata(object):
```

Comment:
What is the point of a class here? Are you sure that a `dict` or `collections.OrderedDict` is not more appropriate?